lib/rubocop/cop/rails/to_s_with_argument.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'set'
```

Comment:
This is unnecessary as it premises a dependency.

Reply:
Thanks, I'll remove this line.

I was wondering about this point as well because this would not a matter of necessity or not, but rather a matter of policy as to who will play the coordinating role, like the contrast between choreography vs. orchestration. I understand that the policy here is to let lib/rubocop.rb in rubocop gem coordinate the dependencies 👌